scripts/change-log-constants.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+const path = require('path');
+
+module.exports = Object.freeze({
```

Comment:
This file was originally going to work in tandem with a new parser class (to reduce the complexity of the main script). I've decided not to include that parser class. Mainly because a lot of this logic is being reused by the port changes script as well. Eventually my goal is to have those two scripts using similar utils - which will mean the refactor will look very different.